.circleci/config.yml

```diff
@@ -31,150 +31,229 @@ jobs:
             source /usr/local/share/virtualenvs/tap-square/bin/activate
             pylint tests -d 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,duplicate-code,no-name-in-module,attribute-defined-outside-init,too-many-statements,cell-var-from-loop,too-many-public-methods,missing-docstring'
       - run:
-          name: 'JSON Validator'
+          name: 'json validator'
```

Comment:
What's up with all the lower case?

Reply:
That was a mistake, I selected all text and hit the button to lowercase everything. if you look at the diff now those lowercases should all be gone.